src/main/java/org/ambraproject/wombat/controller/ArticleController.java

```diff
@@ -776,8 +777,9 @@ private XmlContent getXmlContent(Site site, ArticlePointer articlePointer,
             String citationJournalKey;
             try {
               citationJournalKey = doiToJournalResolutionService.getJournalKeyFromDoi(doi, site);
-            } catch (SolrUndefinedException e) {
+            } catch (SolrUndefinedException | EntityNotFoundException e) {
```

Comment:
I am ok with it. I just want to know which app is throwing error and we can figure out why solr is not good in this case.